RELEASE-NOTES.txt

```diff
@@ -9,6 +9,28 @@ create portable packages of repository content.
 The Vault Command Line Interface aka "vlt" provides a subversion like
 utility to work and develop with repository content.
 
+Changes in Jackrabbit FileVault 3.7.2
+--------------------------------------
+
+Release Notes - Jackrabbit FileVault - Version 3.7.2
+This version requires Java 8
+The OSGi bundles depend on Jackrabbit 2.20.8+ (JCR Commons, SPI, SPI Commons), Oak JR API 1.22.4+, Commons IO 2.7+, Commons Collections 4.1+ and SLF4J 1.7+
+
+Bug
+
+    [JCRVLT-718] - DocumentViewXmlValidator.validateEnd(...) called with wrong nodePath
+    [JCRVLT-721] - Importing content packages with minimum permissions fails
+    [JCRVLT-722] - SyncLog: java.time.temporal.UnsupportedTemporalTypeException: Unsupported field: DayOfMonth
```

Comment:
Please add https://issues.apache.org/jira/browse/JCRVLT-725 which I just merged.

Reply:
And also https://issues.apache.org/jira/browse/JCRVLT-727

Reply:
ack